.github/workflows/tests.yml

```diff
@@ -128,12 +128,12 @@ jobs:
     services:
       postgres:
         # Docker Hub image
-        image: postgres
-        # Provide the password for postgres
+        image: postgres:15.3-alpine3.18
```

Comment:
Agree about explicit versions.

Then I would go for postgres:14-bulleyes (Etc/UTC by default) to prevent issues with locales on alpine (not 100% sure, I'll check). 

I'll add an example in a quick addition